Data/ByteString/Lazy.hs

```diff
@@ -342,12 +342,14 @@ snoc cs w = foldrChunks Chunk (singleton w) cs
 {-# INLINE snoc #-}
 
 -- | /O(1)/ Extract the first element of a ByteString, which must be non-empty.
+--
+-- This is a partial function, please consider using 'uncons' instead.
```

Comment:
```suggestion
-- This is a partial function. Please consider using 'uncons' instead.
```

Maybe even drop the "Please" – to me it sounds like the `bytestring` maintainers would benefit personally if users would use `uncons` instead.

(Dito for the other changes in this module.)